src/index.html

```diff
@@ -11,6 +11,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+      action="#" class="form"
+      data-qa="big" method="get">
```

Comment:
```suggestion
        data-qa="big" 
        method="get"
      >
```
Consider fixing indentations according to the code style guide